awx_collection/requirements.txt

```diff
@@ -0,0 +1,3 @@
+pytz  # for tower_schedule_rrule lookup plugin
+python-dateutil>=2.7.0  # tower_schedule_rrule
+awxkit  # For import and export modules
```

Comment:
heads up to @john-westcott-iv, this is intended to allow import / export modules to work inside of execution environments with no further modification.

I also want to ping @ryanpetrello @matburt just to point out how intensely awesome this is going to be.